### DISCUSSION TITLE
Import track : impossible to import gpx file

### DISCUSSION THREAD

**Discussion by Fredmarre** [osmandapp/OsmAnd#12217] [Q&A]
Hello,

I made an import 2 weeks ago without any issue, but the last tentative is NOK
I am using OSMand Maps on IOS.
I have the following message when importing : import tracks : impossible to import GPS file (translated because I have it in French)
I tried with several GPX files, even the one used 2 weeks ago that was working, and always the same message.
I removed some favorite tracks, I was thinking maybe there is a limited number to be imported, but not change.

Can you help me ?

Thank you
Frédéric

**Comment by vshcherb** [comment]
I think it's only possible to test specific GPX files.

**Reply by Fredmarre** [reply]
Thank you for your reply, but how to make this specific test ?

**Reply by vshcherb** [reply]
Well, it's not clear for which GPX track it works and for which it doesn't. The error could be inside GPX and could be in the program. So far it's hard to guess

**Reply by Fredmarre** [reply]
Ok, clear, also I had tried to re-import previous track, and it was not working ... so I didn't gave up and tried other way to make it, and finally by sending me mail between different mail box, it works....
I don't know why it works only like that now, but it is OK
Thank you very much for your help

**Comment by SportTester** [comment]
the way i import the track is connect to the computer and transfer it to your phone. Then in the app import it however when i use it to navigate its not clean. :(
